Gemfile

```diff
@@ -22,9 +22,12 @@ gem 'rack', '>= 2.0.6'          # Upgrade for security update
 gem 'redcarpet', github: 'vmg/redcarpet' # markdown
 gem 'sass-rails', '~> 6.0'      # Use SCSS for stylesheets
 gem 'uglifier', '>= 1.3.0'      # Use Uglifier as compressor for JavaScript assets
-gem 'will_paginate', '~> 3.2.0' # pagination. Styles: http://mislav.github.io/will_paginate/
+gem 'will_paginate', '~> 3.3'   # Pagination. Styles: http://mislav.github.io/will_paginate/
```

Comment:
`will_paginate` was not behaving well with this new version of ruby, so I upgraded it.